### ISSUE TITLE
LineWidth not applied properly to ErrorBars

### ISSUE THREAD

**Issue by nrontsis** [matlab2tikz/matlab2tikz#960]
Consider the following example:

```
errorbar(0,0,1,'linewidth', 2)
matlab2tikz('standalone', true)
```

This results in the following code in the resulting tikz file:
```
\addplot [color=mycolor1, line width=2.0pt, forget plot]
 plot [error bars/.cd, y dir = both, y explicit]
```

Unfortunately, this code changes only the width of the mean-line. I managed to produce the desired result with the following code:
```
\addplot [color=mycolor1, line width=2.0pt, forget plot]
 plot [error bars/.cd, y dir = both, y explicit,
      error bar style={line width=2pt},
      error mark options={
      rotate=90,
      mark size=5pt,
      line width=2pt
  }
 ]
```

**Comment by okomarov** [comment]
Thanks for reporting this. Currently, we handle error bars at a very basic level. If you would like to look into that and issue a PR, it boils down to [line 1862](https://github.com/matlab2tikz/matlab2tikz/blob/master/src/matlab2tikz.m#L1862). 
The quick and best approach would be to inherit the line options and pass them (somehow) to the error bar/mark options.

**Comment by nrontsis** [comment]
Thanks for the prompt reply and sorry for my delayed response. I have just submitted a PR, let me know what you think.

**Comment by egeerardyn** [comment]
Fixed by #961.
